src/InstrumentationEngine.Lib/ConfigurationLocator.h

```diff
@@ -22,10 +22,12 @@ namespace MicrosoftInstrumentationEngine
         static constexpr const char s_cEnvironmentVariableNameValueSeparator = '=';
         static constexpr const char s_cEnvironmentVariablePathDelimiter = ':';
 #else  // PLATFORM_UNIX
-#if defined(_WIN64)
```

Comment:
>_WIN64 [](http://example.com/codeflow?start=12&length=6)

`_WIN64` supports both X64 and ARM64

[https://docs.microsoft.com/en-us/cpp/preprocessor/predefined-macros?view=msvc-170](https://docs.microsoft.com/en-us/cpp/preprocessor/predefined-macros?view=msvc-170)